app/src/main/java/org/mozilla/focus/utils/FirebaseHelper.java

```diff
@@ -202,5 +208,15 @@ protected Void doInBackground(Void... voids) {
         return map;
     }
 
+    public static void event(String key, String value) {
+        event(key, value, null);
+    }
+
+    public static void event(String key, String value, Bundle extra) {
```

Comment:
```
Maybe event(String method, String object, String value, Bundle extra) {
   final String key = method + object 
}
```
will make your life easier.
In the meanwhile, maybe we need a `-` or `_` to avoid duplicate method + object possibilities? (Quite unlikely though.)